modules/util/config/TrainConfig.py

```diff
@@ -565,7 +567,8 @@ def default_values() -> 'TrainConfig':
         data.append(("mse_strength", 1.0, float, False))
         data.append(("mae_strength", 0.0, float, False))
         data.append(("vb_loss_strength", 1.0, float, False))
-        data.append(("min_snr_gamma", 0, float, False))
```

Comment:
Do we need a config migration defined for this? It will break old configs that have min_snr_gamma set.

Reply:
@Nerogar I'd leave this to you.

Reply:
Done in https://github.com/Nerogar/OneTrainer/pull/212/commits/c74b3e9b574f5575db71be5407dba8b653c42ea8.